pkg/controller/plan/migration.go

```diff
@@ -71,6 +70,7 @@ const (
 	Cutover         = "Cutover"
 	DiskTransfer    = "DiskTransfer"
 	ImageConversion = "ImageConversion"
+	VMCreation      = "VirtualMachineCreation"
```

Comment:
adding this step(and into the pipeline) isn't great. maybe worth thinking on staying on the same step (that would be convert/disk transfer depends on the flow).

Reply:
discussed offline, it's ok to leave it.